tuf/repository_lib.py

```diff
@@ -1445,9 +1477,20 @@ def generate_snapshot_metadata(metadata_directory, version, expiration_date,
       # list these roles found in the metadata directory.
       if tuf.roledb.role_exists(rolename, repository_name) and \
           rolename not in ['root', 'snapshot', 'timestamp', 'targets']:
-        fileinfodict[metadata_name] = get_metadata_versioninfo(rolename,
+
+        length, hashes = securesystemslib.util.get_file_details(
+          os.path.join(metadata_directory, metadata_filename),
+          tuf.settings.FILE_HASH_ALGORITHMS)
```

Comment:
Coding style is to double-indent continuation lines, please fix here and throughout.